src/commands/targeting/__snapshots__/update.test.ts.snap

```diff
@@ -59,10 +59,24 @@ All Users New
     }
   ]
 }
+
+Targeting rules updated.
+
+To enable this feature on this environment, use:
+
+dvc targeting enable feature-key env-key
+
 "
 `;
 
 exports[`Targeting update updates a target in headless mode 1`] = `
 "{\\"_feature\\":\\"647e36gg16d9k4815fi3f97d\\",\\"_environment\\":\\"648b421gg3f682869c0f22f8\\",\\"_createdBy\\":\\"test\\",\\"status\\":\\"inactive\\",\\"updatedAt\\":\\"2023-06-27T03:19:58.541Z\\",\\"targets\\":[{\\"_id\\":\\"647e36gg16d9k2814fc3f15e\\",\\"audience\\":{\\"name\\":\\"All Users\\",\\"filters\\":{\\"operator\\":\\"and\\",\\"filters\\":[{\\"type\\":\\"all\\"}]}},\\"distribution\\":[{\\"_variation\\":\\"647e36gg16c1d4814fe3f962\\",\\"percentage\\":1}]}],\\"readonly\\":false}
+
+Targeting rules updated.
+
+To enable this feature on this environment, use:
+
+dvc targeting enable feature-key env-key
+
```

Comment:
We shouldn't output this hint in headless mode — in headless mode users may want to parse the json output to use elsewhere